php/admin/class-license.php

```diff
@@ -155,12 +167,29 @@ public function license_success_message() {
 		return sprintf( '<div class="notice notice-success"><p>%s</p></div>', esc_html( $message ) );
 	}
 
+	/**
+	 * Admin notice for the license request failing.
+	 *
+	 * This is for when the validation request fails entirely, like with a 404.
+	 * Not for when it returns that the license is invalid.
+	 *
+	 * @return string
+	 */
+	public function license_request_failed_message() {
+		$message = sprintf(
+			/* translators: %s is the email link for support */
+			__( 'There was a problem activating the license, but it may not be invalid. If the problem persists, please <a href="%s">contact support</a>.', 'block-lab' ),
```

Comment:
I know that there's a range of opinion on this, but I'd prefer to put the anchor tag in sprintf variables, like we do elsewhere in the plugin. I know core doesn't always stick to this rule, but I think it makes things much easier for translators. Optional, if you feel strongly against this.

Reply:
Ah, good point. I'll apply that suggestion soon.

Reply:
How does https://github.com/getblocklab/block-lab/pull/406/commits/5c57e18bb39e9fb60991cde3aad4baad70fb3ab2 look?